### ISSUE TITLE
SVG not detected

### ISSUE THREAD

**Issue by ThomasWollmann** [galaxyproject/galaxy#6907]
When uploading SVGs they are always detected as some other dataformat.

**Comment by mvdbeek** [comment]
Correct, they should be detected as xml. Would this be a feature request then ? We can sniff them to SVG, but we probably shouldn't display due to possible XSS attacks.

**Comment by ThomasWollmann** [comment]
I agree that this can face security issues. Maybe one of the solutions in https://digi.ninja/blog/svg_xss.php is suitable for us?
